pkg/util/objects.go

```diff
@@ -16,7 +16,7 @@
 Package manifest provides functions for going between in-memory k8s objects (unstructured.Unstructured) and their JSON
 or YAML representations.
 */
-package manifest
+package util
```

Comment:
See my other comment about overloading the `util` package. `istio.io/operator/pkg/kube` or `istio.io/operator/pkg/kube/ojbect` might be more appropriate here. `K8sObject` could be renamed `Object` and referenced as `kube.Object`, which is consistent with other parts of Istio.

Reply:
Done. I think kube is probably a bit misleading since there's little else in here to do with kube. I've made it object for now, we can easily change it if the package function expands.